osu.Game/Online/API/Requests/Responses/APICountryRankings.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Newtonsoft.Json;
+using osu.Game.Users;
+
+namespace osu.Game.Online.API.Requests.Responses
+{
+    public class APICountryRankings : CountryStatistics
+    {
+        [JsonProperty]
+        public Country Country;
```

Comment:
just add this to `CountryStatistics`. the class name here is hard to understand.